src/main/resources/static/api.yml

```diff
@@ -598,11 +602,15 @@ components:
           type: string
         comments:
           type: string
+        referable:
+          type: boolean
+          default: false
```

Comment:
Just to make sure I've understood correctly, when we upload new courses via the CSV, we default to `false`, but otherwise this field defaults to `true` (in the course type and model)?

I wonder if we should keep these defaults consistent, or was there a reason behind the `false` value here?

Reply:
Agreed, I think the default should be true here.

Reply:
Good spot. Have updated to match the intended functionality - defaulting to `true` across the board.